### DISCUSSION TITLE
REDIS

### DISCUSSION THREAD

**Discussion by tjharman** [Kovah/LinkAce#529] [General]
Hi,

I run a small linkace (love it, thank you) with about 900 links in it.  I can easily setup REDIS as well, but I'm wondering what/how it improves performance?  It's briefly mentioned in the .env settings but that's it.

Is it worth enabling?  My instance at the moment (mysql) seems plenty fast enough.  More speed is never a bad thing though, but I'm curious what exactly REDIS will help with.

Thanks!

**Comment by Kovah** [comment]
Hey, thanks for using LinkAce.
If you are using Redis, user sessions and caches will use it, instead of using the filesystem. It will be faster and reduce filesystem operations, which is always better for your hard drives.

**Reply by tjharman** [reply]
Thank you for the prompt reply.
I've now got REDIS running via a unix socket, very easy.

I assume I can delete the following:

linkace/storage/framework/cache
linkace/storage/framework/sessions

What about


linkace/storage/framework/views ?

Thanks again!

**Reply by Kovah** [reply]
I think views need to be stored in the filesystem, but they usually don't change between updates.
